multigeiger/userdefines-example.h

```diff
@@ -1,67 +1,76 @@
-//====================================================================================================================================
 // User / hardware specific definitions - do NOT edit THIS (example) file.
 //
 // To use them please copy this file to userdefines.h (do not add that file to version control)
 // and then edit userdefines.h as appropriate for your use case / hardware.
-//====================================================================================================================================
 
-// If you use platformio, CPU is defined in platformio.ini and
-// you don't need to change the definition of CPU.
-// Arduino IDE users *must* define CPU here and uncomment one and only one of following CPU-Defines.
+
+// CPU values (board types, DO NOT CHANGE):
```

Comment:
Can we put all these "DO NOT CHANGE" defines in an extra file? 
I think it's dangerous that a user CAN change it.

Reply:
well, if you edit the source (which is always the case except when using a precompiled binary), you can always shoot yourself in the foot.

I can put the constants into yet another header file and #include that there (as I did for some other stuff in userdefines.h).

But then, one has to tell the user their options (constant names), so that is a duplication that has to be kept in sync with the header file.

Reply:
OK agreed.
Then we could cluster this DONT CHANGE together and make  an attention line like ***************************** DONT CHANGE HERE ****************** or similar

Reply:
I just placed the stuff in a way so that you first see your options and directly after that, you configure what you have. Then next options, next configuration. Repeat.

Guess every way has its pros and cons.